library/foundation/src/main/java/com/epmedu/animeal/foundation/input/PhoneInputView.kt

```diff
@@ -0,0 +1,156 @@
+package com.epmedu.animeal.foundation.input
+
+import android.content.res.Configuration.UI_MODE_NIGHT_NO
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement.Center
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.OffsetMapping
+import androidx.compose.ui.text.input.TransformedText
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.epmedu.animeal.base.theme.AnimealTheme
+import com.epmedu.animeal.base.theme.CustomColor.LynxWhite
+
+@Composable
+fun PhoneNumberInputView(
+    modifier: Modifier = Modifier,
+    title: String,
+    onValueChange: (String) -> Unit,
+    value: String
+) {
+    Column(modifier = modifier) {
+        Text(
+            text = title,
+            modifier = Modifier.padding(bottom = 2.dp),
+            style = TextStyle(
+                color = MaterialTheme.colors.onSurface,
+                fontSize = 14.sp
+            )
+        )
+        Row(
+            horizontalArrangement = Center,
+            modifier = Modifier
+                .height(56.dp)
+                .background(
+                    color = LynxWhite,
+                    shape = RoundedCornerShape(12.dp),
+                )
+        ) {
+            Box(
+                modifier = Modifier
+                    .height(56.dp)
+                    .padding(start = 16.dp),
+                contentAlignment = Alignment.Center,
+            ) {
+                Text(
+                    text = "+995",
+                    textAlign = TextAlign.Start,
+                    modifier = Modifier.align(Alignment.Center),
```

Comment:
Alignment is redundant here since you set alignment in the box

Reply:
Removed

Reply:
removed